libpod/container_commit.go

```diff
@@ -74,11 +74,11 @@ func (c *Container) Commit(ctx context.Context, destImage string, options Contai
 	// add it to the resulting image.
 
 	// Entrypoint - always set this first or cmd will get wiped out
-	importBuilder.SetEntrypoint(c.Spec().Process.Args)
+	importBuilder.SetEntrypoint(entryPoint)
```

Comment:
Should we check if entrypoint and command are not "" before we set them? We do for mounts, so it seems like it would make sense

Reply:
i only do that because of the strings split.

Reply:
Probably safest to ignore them if they're "", I think they might be in some cases where the user didn't set one or the other

Reply:
fixed